MediaBrowser.Controller/Lyrics/LyricResponse.cs

```diff
@@ -0,0 +1,22 @@
+#nullable disable
+
+using System.Collections.Generic;
+
+namespace MediaBrowser.Controller.Lyrics
+{
+    /// <summary>
+    /// LyricResponse model.
+    /// </summary>
+    public class LyricResponse
+    {
+        /// <summary>
+        /// Gets or sets MetaData.
+        /// </summary>
+        public IDictionary<string, object> MetaData { get; set; }
```

Comment:
```suggestion
        public IDictionary<string, object> Metadata { get; set; }
```
For consistency with other models